protocol/src/test/java/org/opensearch/sql/protocol/response/format/ErrorFormatterTest.java

```diff
@@ -16,7 +17,7 @@ class ErrorFormatterTest {
   @Test
   void htmlEscaping_should_disabled() {
     assertEquals(
-        "{\n" + "  \"request\": \"index=test\"\n" + "}",
+        format("{\n" + "  \"request\": \"index=test\"\n" + "}"),
```

Comment:
Should be there `%n` too?

Reply:
Good catch I reverted this file. As per @joshuali925's suggestion, I will not be doing replaceAll in ErrorFormatter, as this could affect user entered input. Reverted this file to initial state, as I am no longer doing anything in the ErrorFormatter itself.